### ISSUE TITLE
Does this support sync?

### ISSUE THREAD

**Issue by didip** [rlmcpherson/s3gof3r#51]
Similar to `s3cmd sync`?

If not, can I make a feature request to have sync?

**Comment by rlmcpherson** [comment]
No, there is no sync feature as the common use case is streaming data that is modified in the stream such as `tar -czf - <my_dir/> | gof3r put...` where syncing does not apply, but it could be added to the cli without any changes to the library.
